### PR TITLE
Support update SQL & fix precision problem

### PR DESCRIPTION
**Pull request by lixupeng**
I do two minor changes:

1) use executeUpdate() to run those CREATE, DROP, INSERT.... statements in trace files. 

2) use BigDecimal to solve the double error #50

### CODE REVIEW

File: src/main/java/edu/cmu/cs/db/peloton/test/common/TestContainer.java [hunk 1]
```diff
@@ -60,7 +67,16 @@ private static void testEqual(ResultSet truth, ResultSet target) throws SQLExcep
     private static List<Object> getRow(ResultSet resultSet, int columnCount) throws SQLException {
         List<Object> row = new ArrayList<>();
         for (int i = 1; i <= columnCount; i++) {
-            row.add(resultSet.getObject(i));
```
Comment (tli2) [inline src/main/java/edu/cmu/cs/db/peloton/test/common/TestContainer.java:63]: This probably won't work as BigDecimal::valueOf uses the string representation of a double as the canonical value and that might already have round-off errors. You should use a delta comparison instead.

**Review by tli2 — CHANGES_REQUESTED** [review]

**Comment by apavlo** [comment]
@lixupeng I would not check in the SQLite traces into the repository. They are going to be huge. You can download them from the peloton website.

The setup script should also probably be in the regular peloton repository.

**Comment by apavlo** [comment]
@lixupeng You need to expunge the sqlite trace file completely from your PR. Otherwise it bloats the size of the repo because it's still in the history:

https://help.github.com/articles/removing-sensitive-data-from-a-repository/

**Comment by lixupeng** [comment]
@apavlo ok, I've fixed it.

**Pull request merged**